pkg/cloud/services/securitygroup/securitygroups_test.go

```diff
@@ -439,17 +440,115 @@ func TestDeleteSecurityGroups(t *testing.T) {
 				},
 			},
 			expect: func(m *mock_ec2iface.MockEC2APIMockRecorder) {
-				m.DescribeSecurityGroupsPages(gomock.Any(), gomock.Any()).Return(nil)
+				m.DescribeSecurityGroupsPages(gomock.AssignableToTypeOf(&ec2.DescribeSecurityGroupsInput{}), gomock.Any()).Return(nil)
```

Comment:
What is the difference of using this:
`				m.DescribeSecurityGroupsPages(gomock.Any(), gomock.Any()).Return(nil)
`and 
`			m.DescribeSecurityGroupsPages(gomock.AssignableToTypeOf(&ec2.DescribeSecurityGroupsInput{}), gomock.Any()).Return(nil)`

Reply:
I saw this change in a couple of other open PRs too but this is just longer without an benefit.
Nil input is same with empty in AWS SDK calls.

Reply:
This is nit.

/approve